docs/networking/proxies/nginx.md

```diff
@@ -397,7 +397,11 @@ These are provided as an alternative to the generic Nginx error page, are shared
 
 ### Default site
 
-By default, Dokku will route any received request with an unknown HOST header value to the lexicographically first site in the nginx config stack. If this is not the desired behavior, you may want to add the following configuration to the global nginx configuration.
+By default, Dokku will route any received request with an unknown HOST header value to the lexicographically first site in the nginx config stack.
+
+> Note that some versions of Nginx may create a default site when installed. This site is simply a static page which says "Welcome to Nginx", and if this default site is enabled, Nginx will not route any requests with an unknown HOST header to Dokku. If you want Dokku to receive all requests, disable this default site by removing the `/etc/nginx/sites-enabled/default` file, and then restart Nginx.
```

Comment:
Mind changing the prefix to `> Warning:` instead of `> Note`? It'll render like a warning :)

Also, maybe add the exact command users should run, like so:

```
> Warning: that some versions of Nginx may create a default site when installed. This site is simply a static page which says "Welcome to Nginx", and if this default site is enabled, Nginx will not route any requests with an unknown HOST header to Dokku. If you want Dokku to receive all requests, run the following commands:
>
> ```
> sudo rm -f /etc/nginx/sites-enabled/default
> dokku nginx:stop
> dokku nginx:start
> ```
```

This will make the work less ambiguous for users.